portlets/src/main/frontend/src/apps/components/leaderboard/Leaderboard.vue

```diff
@@ -24,11 +24,17 @@
             <div class="col">
                 <div role="toolbar" class="btn-toolbar" aria-label="Toolbar with button groups and dropdown menu">
                     <div id="app" role="group" class="btn-group">
-                        <button type="button" class="btn btn-secondary" @click="activeBtn = 'btn1';selectedPeriod = 'WEEK';loadCapacity=10"
+                        <button @click="activeBtn = 'btn1'selectedPeriod = 'WEEK';loadCapacity=10"
```

Comment:
This expression is wrong, Didn't it cause the fail of the build. The same problem in lines 31 and 35

Reply:
ok